### ISSUE TITLE
Compute ZRX / BNB in telliot

### ISSUE THREAD

**Issue by Spuddyminer** [tellor-io/telliot#481]
To avoid binance api problems, take (ZRX/USD) / (BNB/USD) as the value to submit.

**Comment by krasi-georgiev** [comment]
I thought form the discussion on discord that we need to make a plan how to remove this ID all together.

**Comment by themandalore** [comment]
We did, but for robustness of the system, we should hav ethe ability to calculate pairs from others pairs as an extra check 
    On Friday, August 6, 2021, 04:44:30 AM EDT, Krasimir Georgiev ***@***.***> wrote:  
 
 


I thought form the discussion on discord that we need to make a plan how to remove this ID all together.


—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by krasi-georgiev** [comment]
Yep good idea. There might be other pairs with a single direct api source
